src/pages/api/recipes/index.ts

```diff
@@ -0,0 +1,123 @@
+import { DEFAULT_PAGE_SIZE } from "@/constants";
+import type { NextApiRequest, NextApiResponse } from "next";
+import prisma, {
+  calculateFavoriteStats,
+  calculateTotalStats,
+} from "@/lib/prisma";
+import { createRedisInstance, getCachedDataFromRedis } from "@/lib/ioredis";
+import { REDIS_RECIPE_STATS_KEY } from "@/constants";
+import { ErrorResponse, RecipesResponse } from "@/types";
+
+const redis = createRedisInstance();
```

Comment:
should we create a common redis instance to be imported? I saw that we have two instances here and [here](https://github.com/aiven/nextjs-netlify/pull/2/files#diff-59ab858d61b08f82c4e6ce33fd4312182442be07a72d45ae30e3d95dd6508e21R11)